### PR TITLE
Use pretty_tox to run tests

### PR DESCRIPTION
**Pull request by jd**
So we can have progress output like it's 2017

### CODE REVIEW

**Comment by sileht** [comment]
Can we just to 2018 right now ?

**Comment by jd** [comment]
Pull-request updated, HEAD is now 0f8f55ccbafd243a2f156a14e22ed3131a7dc47e

**Comment by sileht** [comment]
I think we should switch to pytest instead of testr, but I have checked pytest is available only in rhel8.

File: setup.cfg [hunk 1]
```diff
@@ -82,6 +82,7 @@ test =
   python-subunit>=0.0.18
   testrepository>=0.0.18
   testtools>=1.4.0
+  os-testr
```
Comment (chungg) [inline setup.cfg]: why?

Comment (jd) [reply setup.cfg]: It provides `subunit-trace` :(

Comment (chungg) [reply setup.cfg]: dammit

**Pull request merged**